tasks/queue-pdrs/schemas/config.json

```diff
@@ -6,13 +6,17 @@
     "collection",
     "provider",
     "queueUrl",
-    "parsePdrMessageTemplateUri"
+    "parsePdrWorkflow",
+    "stackName",
+    "internalBucket"
   ],
   "additionalProperties": false,
   "properties": {
     "collection": { "type": "object" },
     "provider": { "type": "object" },
     "queueUrl": { "type": "string" },
-    "parsePdrMessageTemplateUri": { "type": "string" }
+    "parsePdrWorkflow": { "type": "string" },
```

Comment:
Yes, this naming is much more logical

Reply:
It's not just naming, it's a different parameter now, workflow name instead of s3 URI.

Reply:
Yeah, makes sense